src/components/shared/contact-column.json

```diff
@@ -0,0 +1,32 @@
+{
+  "collectionName": "components_shared_contact_columns",
+  "info": {
+    "displayName": "ContactColumn"
+  },
+  "options": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "text": {
+      "type": "string",
+      "required": true,
+      "maxLength": 42
+    },
+    "label": {
+      "type": "string",
+      "required": true
+    },
+    "icon": {
+      "type": "relation",
+      "relation": "oneToOne",
```

Comment:
> Suggestion: Probably it should be `oneToMany` relation because we might have multiple icons, right? Also, probably, it would be better to have `icon` => `icons`.

Yes, agree. Good suggestion. The changes has been applied